### PR TITLE
Add captured api modal

### PR DESCRIPTION
**Pull request by RaiVaibhav**
Implement `capturedapi` Modal for the https://github.com/appknox/mycroft/pull/1065

### CODE REVIEW

File: app/locales/ja/translations.js [hunk 1]
```diff
@@ -507,7 +507,7 @@ export default {
     },
     "runAPIScan": {
       "title": "API診断の実行",
-      "description": "API診断を開始するには、動的診断も実行する必要があります",
+      "description": "キャプチャしたAPIスキャンを実行する",
```
Comment (yashwin) [inline app/locales/ja/translations.js]: Did you google translate here?

File: app/models/capturedapi.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+    is_active: DS.attr('boolean'),
```
Comment (yashwin) [inline app/models/capturedapi.js]: This should be `isActive`

File: app/components/dynamic-scan.js [hunk 3]
```diff
@@ -128,6 +134,14 @@ export default Component.extend({
     openRunDynamicScanModal() {
       this.set("showRunDynamicScanModal", true);
     },
+    
+    closeCapturedApiModal(){
+      this.set("showAPIScanModal", false);
+      this.set("showAPIURLFilterScanModal", false);
```
Comment (yashwin) [inline app/components/dynamic-scan.js]: This isn't required, right?

File: app/components/dynamic-scan.js [hunk 4]
```diff
@@ -128,6 +134,14 @@ export default Component.extend({
...
+      this.set("showRunDynamicScanModal", false);
```
Comment (yashwin) [inline app/components/dynamic-scan.js]: This also isn't required

File: app/components/captured-apis/template.emblem [hunk 5]
```diff
@@ -0,0 +1,11 @@
+if hasObjects
+  each sortedObjects as |capturedapi|
+    = capturedapis-overview file=file capturedapi=capturedapi
```
Comment (yashwin) [inline app/components/captured-apis/template.emblem]: Can you rename this component as `captured-api-overview`

File: app/locales/ja/translations.js (hunk 1 above)
Comment (RaiVaibhav) [reply app/locales/ja/translations.js]: Yes

File: app/components/dynamic-scan.js (hunk 3 above)
Comment (RaiVaibhav) [reply app/components/dynamic-scan.js]: yes, forgot to remove :+1:

File: app/locales/ja/translations.js (hunk 1 above)
Comment (yashwin) [reply app/locales/ja/translations.js]: Ok, for now use the english text only

**Comment by Praseetha-KR** [comment]
##### CHANGELOG
- Isolated api-scan component from dynamic-scan component, refactor UI for both
- Moved API URL filters to dynamic-scan settings
- API scan modal with captured apis list

**Pull request merged**